agent/api/task/task_linux.go

```diff
@@ -262,7 +262,8 @@ func (task *Task) GetCredentialSpecResource() ([]taskresource.TaskResource, bool
 
 // initializeExecCommandAgentResources specifies the necessary volumes and mount points in all of the task containers in order for the
 // exec agent to run upon container start.
-func (task *Task) initializeExecCommandAgentResources(cfg *config.Config) error {
+// TODO: [ecs-exec] Should we validate the ssm agent binaries & certs are valid and fail here if they're not? (bind mount will succeed even if files don't exist in host)
```

Comment:
feels like this should be part of agent capability advertisement.

Reply:
Is there any action item here? not exactly sure what this means :)

Reply:
we'll do some planning around this -- I think for now this is ok as a TODO.